examples/getting_started_uuid/README.md

```diff
@@ -0,0 +1,8 @@
+# butane `getting_started_uuid` example
```

Comment:
I'm not quite following why we need a whole separate example that's mostly a copy of the other one except it makes use of deriving a FieldType? Why not one of
1. Just a test using this functionality
2. Extend one of the models in the getting_started example with a new field showcasing this
3. A new wholly separate (and probably smaller, without several separate commands) example showcasing this.

Reply:
Your points are well taken. I like this suggestion:

> I could change this example to be "newtype_uuid" ; happy to drop all of the src/bin.

I think that preserves the benefits of this being an example while being clearer on its purpose and reducing duplication